clients/out/test/resources/log4j.properties

```diff
@@ -12,10 +12,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-log4j.rootLogger=OFF, stdout
+log4j.rootLogger=TRACE, stdout
```

Comment:
Perhaps didn't intend to checkin these changes? Actually I dont know why this file is even in github. Looking at the history, it has got me as the author, but it is not from my PR. Perhaps @ijuma checked it in inadvertently?

Reply:
That's odd. https://github.com/apache/kafka/pull/4172

Reply:
I have merged #4172.